api/chaos/experimentation/v1/create_experiment_data.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package clutch.chaos.experimentation.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/lyft/clutch/backend/api/chaos/experimentation/v1;experimentationv1";
+
+// The data used as an input for experiment creation.
+message CreateExperimentData {
+  // The unique identifier of experiment run that's created as part of the experiment creation process.
```

Comment:
Saying that this is created as part of the creation process seems odd as the introductionary sentence here, why would it be a parameter in that case?

Reply:
experiment run is created as part of the process - a caller just provides its identifier (if any).